### ISSUE TITLE
Should <label> have a transparent content model?

### ISSUE THREAD

**Issue by travisleithead** [w3c/html#266]
Moved from Bugzilla: https://www.w3.org/Bugs/Public/show_bug.cgi?id=23561
@asurkov, @komputist

Thread ends with a suggestion to make `<label>` have a transparent content model (to allow `<div>` or `<p>`).

Need to check the interop status of such a change.

**Comment by Wes1262** [comment]
yes. exactly like `<a>`

**Comment by stevefaulkner** [comment]
It's been a year and no tests have been forthcoming. Closing this for now, please re-open when tests are available.

**Comment by chaals** [comment]
One test in Yandex, Safari and Firefox browsers of [two paragraphs in a label](http://chaals.github.io/testcases/label-block-content-manual.html) shows that clicking on the works fine, but navigating to bits of the label and clicking on them with VoiceOver active fails in Safari. There is also a [test with a paragraph including a link in the label](http://chaals.github.io/testcases/label-link-content-manual.html) to try. Systematic results would be nice.
